Pages/Index.cshtml

```diff
@@ -74,30 +75,30 @@
 }
 
 <h3 class="text-center w-100 text-dark h3 mt-5 mb-2">Safest Neighborhoods In Cincinnati</h3>
-<div class="row g-3">
+<div class="row justify-content-center g-3">
     <div class="col-4 ">
-        <img src="https://i0.wp.com/haushomemagazine.com/wp-content/uploads/2021/04/My-Post-6.jpg?w=1200&ssl=1" class="zoom-hover rounded-top" alt="image of place" />
+        <img src="https://i0.wp.com/haushomemagazine.com/wp-content/uploads/2021/04/My-Post-6.jpg?w=1200&ssl=1" class="zoom-hover rounded-top" alt=">Hyde Park, Cincinnati" />
```

Comment:
Modified alt attributes value to more accurate descriptive for images.

Reply:
Very good.  Consider describing Hyde Park, briefly.  What would a user see in this image?